indra/tests/test_grounding_resources.py

```diff
@@ -0,0 +1,44 @@
+import os
+import csv
+from indra.statements.validate import validate_db_refs, validate_ns
+from indra.preassembler.grounding_mapper import default_grounding_map
+from indra.preassembler.grounding_mapper import default_misgrounding_map
+
+
+# Namespaces that are not currently handled but still appear in statements
+exceptions = ['CLO']
```

Comment:
With the modification to the grounding to CLO I proposed above, this won't be necessary.